### ISSUE TITLE
[1.1.0.6+] WSDL generation using XmlSerializer is broken

### ISSUE THREAD

**Issue by niker** [DigDes/SoapCore#600]
When I update my project to 1.1.0.6 or 1.1.0.7, the WSDL generation fails to link the request object properly. All I get in a client is an empty SoapEnvelope. When I downgrade to 1.1.0.5, everything just works.

WSDL generated in 1.1.0.5:
```
<wsdl:message name="Partner_StartFinancing_InputMessage">
  <wsdl:part name="parameters" element="tns:StartFinancing" />
</wsdl:message>
```
In 1.1.0.6+:
```
<wsdl:message name="Partner_StartFinancing_InputMessage" />
```

My startup:
```
endpoints.UseSoapEndpoint<PartnerContract>(options =>
{
    options.Binding = new BasicHttpBinding();
    options.Path = "/Soap/Partner.svc";
    options.CaseInsensitivePath = true;
    options.SoapSerializer = SoapSerializer.XmlSerializer;
});
```

Contract:
```
[OperationContract]
public async Task<ResultInfo> StartFinancingAsync(StartFinancingRequest request)
```

Request:
```
[XmlRoot(ElementName = "StartFinancingRequest")]
public class StartFinancingRequest
{
    [XmlElement]
    public string DealerNumber { get; set; }
}
```

Any idea if there was an undocumented breaking change concerning XmlSerializer in 1.1.0.6?

**Comment by ArtsiomKazlouski** [comment]
Same for me

**Comment by martinp999** [comment]
Same for me.  In my case, this is the first time that I have used the package so took forever to realise that I wasn't doing anything wrong.  Doh! ;-)

**Comment by aldycool** [comment]
Same here, confirmed that downgrading to 1.1.0.5 fixed this issue.

**Comment by IvanLeonenko** [comment]
I added a fix in this PR - https://github.com/DigDes/SoapCore/pull/638

**Comment by kotovaleksandr** [comment]
fixed in #638, thanks!
